aten/src/ATen/mps/MPSDevice.mm

```diff
@@ -97,11 +97,15 @@ static inline MTLLanguageVersion getMetalLanguageVersion(const id<MTLDevice>& de
   static bool _macos_13_1_plus = [mpsCD instancesRespondToSelector:@selector(
     sampleGridWithSourceTensor:coordinateTensor:layout:normalizeCoordinates:relativeCoordinates:alignCorners:paddingMode:samplingMode:constantValue:name:)] == YES;
   static bool _macos_13_2_plus = [mpsCD instancesRespondToSelector:@selector(convolution3DWithSourceTensor:weightsTensor:descriptor:name:)] == YES;
+  static bool _macos_13_3_plus = NO;
+  if (@available(macOS 13.3, *))
+    _macos_13_3_plus = YES;
```

Comment:
This would never return True, as library availability level is equal to host executable availability level, which if I'm not mistaken is setup for 11.0 for all Python builds shipped by Anaconda

Reply:
Agreed, we didn't see this issue locally. But we will switch it out to a ObjC selector.